### PR TITLE
display category tags when selecting content

### PR DESCRIPTION
**Pull request by mconnell**
Adds tags/labels to each source item.

Before:
screen shot 2015-07-27 at 11 48 53

After:
screen shot 2015-07-27 at 11 44 18

### CODE REVIEW

File: app/views/admin/feed_source_items/_source_item_checkboxes.html.erb [hunk 1]
```diff
@@ -6,5 +6,10 @@
       <br>
       <%= link_to_if source_item.published?, source_item.url, source_item.url, target: '_blank' %>
     <% end %>
+    <div>
```
Comment (neoeno) [inline app/views/admin/feed_source_items/_source_item_checkboxes.html.erb]: You can omit this `div` I think

Comment (neoeno) [inline app/views/admin/feed_source_items/_source_item_checkboxes.html.erb]: (scratch that for now)

**Comment by neoeno** [comment]
Also when rootin around I noticed this:

image

**Comment by mconnell** [comment]
That's an issue which is resolved in the other PR I have open

File: app/views/admin/feed_source_items/_source_item_checkboxes.html.erb (hunk 1 above)
Comment (neoeno) [inline app/views/admin/feed_source_items/_source_item_checkboxes.html.erb]: Actually it is needed after all!

**Comment by neoeno** [comment]
:+1:

**Comment by darokel** [comment]
:+1:

**Pull request merged**